etc/findbugsExclude.xml

```diff
@@ -0,0 +1,133 @@
+<!--
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<FindBugsFilter>
+  <Match>
+    <Class name="io.openmessaging.benchmark.driver.pulsar.config.PulsarClientConfig$AuthenticationConfiguration" />
+    <Bug pattern="UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD" />
+  </Match>
+  <Match>
+    <Class name="io.openmessaging.benchmark.driver.jms.JMSBenchmarkConsumer" />
+    <Bug pattern="EI_EXPOSE_REP2" />
+  </Match>
+  <Match>
+    <Class name="io.openmessaging.benchmark.driver.jms.JMSBenchmarkProducer" />
+    <Bug pattern="EI_EXPOSE_REP2" />
+  </Match>
+  <Match>
+    <Class name="io.openmessaging.benchmark.driver.jms.JMSBenchmarkTransactionProducer" />
+    <Bug pattern="EI_EXPOSE_REP2" />
+  </Match>
+  <Match>
+    <Class name="io.openmessaging.benchmark.driver.kafka.KafkaTopicCreator" />
+    <Bug pattern="RV_RETURN_VALUE_IGNORED_BAD_PRACTICE" />
+  </Match>
+  <Match>
+    <Class name="io.openmessaging.benchmark.driver.kop.config.PulsarConfig" />
+    <Bug pattern="UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD" />
+  </Match>
+  <Match>
+    <Class name="io.openmessaging.benchmark.driver.pravega.config.PravegaClientConfig" />
+    <Bug pattern="UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD" />
+  </Match>
+  <Match>
+    <Class name="io.openmessaging.benchmark.driver.rabbitmq.RabbitMqBenchmarkProducer" />
+    <Bug pattern="RV_RETURN_VALUE_OF_PUTIFABSENT_IGNORED" />
+  </Match>
+  <Match>
+    <Class name="io.openmessaging.benchmark.driver.rabbitmq.RabbitMqBenchmarkConsumer" />
+    <Bug pattern="EI_EXPOSE_REP2" />
+  </Match>
+  <Match>
+    <Class name="io.openmessaging.benchmark.driver.rabbitmq.RabbitMqBenchmarkProducer" />
+    <Bug pattern="EI_EXPOSE_REP2" />
+  </Match>
+  <Match>
+    <Class name="io.openmessaging.benchmark.driver.bookkeeper.DlogBenchmarkConsumer" />
+    <Bug pattern="EI_EXPOSE_REP2" />
+  </Match>
+  <Match>
+    <Class name="io.openmessaging.benchmark.driver.bookkeeper.DlogBenchmarkProducer" />
+    <Bug pattern="EI_EXPOSE_REP2" />
+  </Match>
+  <Match>
+    <Class name="io.openmessaging.benchmark.driver.bookkeeper.stats.StatsLoggerAdaptor" />
+    <Bug pattern="EI_EXPOSE_REP2" />
+  </Match>
+  <Match>
```

Comment:
please cleanup this file and remove all content that isn't relevant to this project.